bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/verify/steps/Step.scala

```diff
@@ -0,0 +1,18 @@
+package uk.ac.wellcome.platform.archive.bagverifier.verify.steps
+
+trait Step {
+  case class BagVerifierError(
```

Comment:
I might but this in a companion object so they are referenceable outside this trait without instantiating it.

Reply:
Good suggestion, doing that.

(At which point the Step trait becomes so small it collapses in on itself, so it's out.)